src/adapters/deployer/index.ts

```diff
@@ -19,66 +19,68 @@ export function createDeployerComponent(
       try {
         const exists = await components.storage.exist(entity.entityId)
 
-        if (!exists) {
-          await components.downloadQueue.onSizeLessThan(1000)
+        const isSnsEntityToSend =
+          !!(entity.entityType === 'scene' || entity.entityType === 'wearable' || entity.entityType === 'emote') &&
```

Comment:
```suggestion
          (entity.entityType === 'scene' || entity.entityType === 'wearable' || entity.entityType === 'emote') &&
```

Reply:
`components.sns.arn` could be a string or undefined, so with `!!` we are foncing a boolean

Reply:
WDYT about using the `!!` just in the `components.sns.arn`?